ci/pipelines/pay-stubs.yml

```diff
@@ -0,0 +1,74 @@
+---
+resource_types:
+  - name: cf-cli
+    type: registry-image
+    source:
+      repository: nulldriver/cf-cli-resource
+
+resources:
+  - name: pay-stubs-pipeline
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: pp-9582/move-pay-stubs-to-paas
+      paths:
+        - ci/pipelines/pay-stubs.yml
+
+  - name: pay-stubs-manifest
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: pp-9582/move-pay-stubs-to-paas
+      paths:
+        - ci/pay_stubs
+
+  - name: paas
+    type: cf-cli
+    icon: cloud-upload-outline
+    source:
+      api: https://api.cloud.service.gov.uk
+      org: govuk-pay
+      space: build
+      username: ((cf-username))
+      password: ((cf-password))
+      cf_cli_version: 7
+
+  - name: stubs-dockerhub
```

Comment:
Is there a reason we're using Dockerhub rather than the stubs ECR repository? (We build & push stubs to ECR in the [e2e-helpers pipeline](https://pay-cd.deploy.payments.service.gov.uk/teams/pay-dev/pipelines/e2e-helpers))

Reply:
While it's the same image in Dockerhub, it'll be quicker for Concourse to fetch the image from ECR (no rate limiting etc).

Reply:
Concourse isn't the thing that needs to fetch the image, it's actually PaaS that needs to pull it which doesn't have access to our ECR so I think we have to pull it from dockerhub

Reply:
Makes sense!

Reply:
Just had a look and it requires an aws access key id and secret access key to do so, which means we would have to supply it with user credentials which feels painful, or orchestrate generating some with STS and passing them (which is much more secure since they would be temporary and we could set a timeout on them of 5 minutes or something).

Either way I think it's worth committing this as-is and changing it on it's own since it would be more complex.

Reply:
See https://docs.cloudfoundry.org/devguide/deploy-apps/push-docker.html#ecr